src/testing/cluster/manifest_checker.zig

```diff
@@ -54,15 +54,16 @@ pub fn ManifestCheckerType(comptime Forest: type) type {
             for (0..constants.lsm_levels) |level| {
                 checksum_stream.add(std.mem.asBytes(&level));
 
-                inline for (Forest.tree_id_range.min..Forest.tree_id_range.max + 1) |tree_id| {
+                inline for (Forest.tree_id_range.min..Forest.tree_id_range.max + 1) |tree_id_u16| {
```

Comment:
Intentionally avoiding `enum.values` here such that the verification code doesn't rely on the order of filds in an enum, and does a stupid numeric loop.